### PR TITLE
Modified create video method to use aws backend code

### PR DESCRIPTION
**Pull request by OussamaSaoudi**
Modified the createVid method used to upload a video to
the data lake to perform a get request to recieve an expiring link
to upload to s3 bucket. This link will allow the front end to directly
upload the user's video to the datalake.

Related to: #49
Depends on: orate-io/orate-io-backend#32

Signed-off-by: Oussama Saoudi <oussama.sa216@gmail.com>

### CODE REVIEW

File: src/components/Signup.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+          name = 'passwordResub'
+        />
+      </div>
+
+      <button type = "submit">sign up</button>
+    </form>
+  )
+}
+
+export default Signup
```
Comment (Yacine-Saoudi) [inline src/components/Signup.js:84]: newline

File: src/components/Video.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+          </button>
+        </div>
+
+      </div>
+    )
+  }
+}
+
+
+export default Video
```
Comment (Yacine-Saoudi) [inline src/components/Video.js:59]: newline

File: src/store.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+})
+
+const store = createStore(
+  combinedReducer,
+  composeWithDevTools(
+    applyMiddleware(thunk)
+  )
+)
+
+export default store
```
Comment (Yacine-Saoudi) [inline src/store.js:24]: newline

File: src/components/Signup.js (hunk 1 above)
Comment (OussamaSaoudi) [reply src/components/Signup.js:84]: This is not rly my code. It's like this due to my base being your commit. Many of these changes will disappear once your changes are merged.

**Review by Yacine-Saoudi — APPROVED** [review]
LGTM!!1

**Pull request merged**